instrumentation/apache-dubbo/apache-dubbo-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachedubbo/v2_7/DubboInstrumentationModule.java

```diff
@@ -32,6 +32,11 @@ public DubboInstrumentationModule() {
     return hasClassesNamed("org.apache.dubbo.rpc.Filter");
   }
 
+  @Override
+  public boolean isLibraryInstrumentationClass(String className) {
+    return className.startsWith("io.opentelemetry.instrumentation.apachedubbo.v2_7");
```

Comment:
Can this be replaced with `this.getClass().getPackage().getName()` and moved to superclass?

Reply:
And remove `.javaagent`, yes.

I thought about this, but unfortunately this would break vendor specific instrumentation modules that extend ones from this repo. I think that @pavolloffay actually had encountered this problem before.

On the other hand, if we officially recommended composition instead of extending (like I suggested [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1677#issuecomment-734949495)) then we could make all modules final and reap benefits of our package naming convention.